pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -303,6 +306,12 @@ func desiredLoadBalancerService(ci *operatorv1.IngressController, deploymentRef
 			// LB relies on iptable rules kube-proxy puts in to send traffic from the VIP node to the cluster
 			// If policy is local, traffic is only sent to pods on the local node, as such Cluster enables traffic to flow to  all the pods in the cluster
 			service.Spec.ExternalTrafficPolicy = corev1.ServiceExternalTrafficPolicyTypeCluster
+		case configv1.PowerVSPlatformType:
```

Comment:
Please combine the cases.
```suggestion
		case configv1.PowerVSPlatformType, configv1.IBMCloudPlatformType:
```

Reply:
Sure will combine the cases here